submissions/mxmgny/a-tiny-js-world/index.js

```diff
@@ -0,0 +1,43 @@
+let cat = {
+    appearance: "cat",
+    name: "Garfield",
+    legs: 4,
+    hands: 0,
+    gender: "male",
+    shout: "I love lazagna!",
+    friends:['Selina', 'Jeremy']
```

Comment:
Pay attention to code style in all file. To avoid making a choice every time you're about to write a string, you should pick one style and stick with it. If you still making up your mind - **pick single quotes**, they are more common. Take a look at this [article](https://bytearcher.com/articles/single-or-double-quotes-strings-javascript) to know, how to correctly combine different parenthases in especially cases.

Reply:
Done